src/components/Card/_card.scss

```diff
@@ -3,11 +3,20 @@
 
 .card {
   &--title {
-    display: flex;
-    @include type-style('productive-heading-01');
-    overflow: hidden;
-    text-overflow: ellipsis;
-    white-space: nowrap;
+    width: 100%;
+    padding-right: $spacing-05;
+    &-wrapper {
+      @include type-style('productive-heading-01');
+      overflow: hidden;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+    }
+    &-tooltip {
+      max-width: 100%;
```

Comment:
Instead of this, we can use min-width on children of flex children that need to be truncated 

https://css-tricks.com/flexbox-truncated-text/#article-header-id-3